tensorflow_addons/metrics/hamming.py

```diff
@@ -49,7 +53,12 @@ def hamming_distance(actuals, predictions):
     return ham_distance
 
 
-def hamming_loss_fn(y_true, y_pred, threshold, mode):
+def hamming_loss_fn(
+    y_true: TensorLike,
+    y_pred: TensorLike,
+    mode: str,
+    threshold: Union[FloatTensorLike, None],
```

Comment:
Be careful, you changed the parameters' order multiple times in this pull request. It's going to break some user code.

We can't change the public API because we give strong backward compatibilities garantees.

Reply:
Thanks for the reminder, @gabrieldemarmiesse.